OBSDOCS-3318 [DOC] Documentation for Perses Dashboard (#111992)
openshift/openshift-docs: modules/coo-perses-dashboard-architecture.adoc

The Perses dashboards feature consists of the following components:

Cluster Observability Operator (COO)::
The COO installs and manages the Perses Operator and its CRDs automatically. The Perses Operator runs continuously after COO installation and does not require the monitoring plugin to be enabled.

Perses Operator::
The Perses Operator manages the lifecycle of Perses server instances and reconciles `PersesDashboard`, `PersesDatasource`, and `PersesGlobalDatasource` custom resources. It synchronizes resources across all Perses servers in the cluster.

Monitoring UIPlugin::
The UIPlugin resource with `spec.monitoring.perses.enabled: true` installs the Perses UI and creates a Perses server instance. It adds the *Observe* > *Dashboards (Perses)* menu to the {product-title} web console.

Custom resource definitions (CRDs)::
The COO installs three CRDs:
+
* `PersesDashboard`: Namespace-scoped dashboard definitions.
* `PersesDatasource`: Namespace-scoped datasource configurations.
* `PersesGlobalDatasource`: Cluster-scoped datasources accessible from all namespaces.

The following hierarchy shows how components relate to each other:

> [source,text]
> ----
> Cluster Observability Operator
> └── UIPlugin (Monitoring with Perses enabled)
>     └── Perses Operator
>         ├── PersesDashboard CRs
>         ├── PersesDatasource CRs
>         └── PersesGlobalDatasource CRs
> ----

Reviewer: ```suggestion
----
Cluster Observability Operator
└── UIPlugin (Monitoring with Perses enabled)
│  └── Dashboards (Perses) UI
└── Perses Operator
    ├── PersesDashboard CRs
    ├── PersesDatasource CRs
    └── PersesGlobalDatasource CRs
----
```

== Namespace and project mapping

The Perses Operator maps Kubernetes namespaces to Perses projects. When you create a `PersesDashboard` or `PersesDatasource` in a namespace, the operator synchronizes that resource into the corresponding Perses project across all Perses servers in the cluster.

> This mapping enables RBAC at the namespace level. A user with permissions in `my-namespace` can only view or edit dashboards within that namespace.

Reviewer: ```suggestion
This mapping enables namespace-level RBAC using Perses Roles and RoleBindings. A user with these Perses permissions in my-namespace can only view or edit dashboards within that namespace.
```

== Datasource resolution

When a panel queries data, Perses resolves the datasource in the following order:

> . A datasource explicitly referenced in the panel query.

Reviewer: This is not supported in OpenShift
```suggestion
```

> . The default namespace-scoped `PersesDatasource` in the same namespace as the dashboard.

Reviewer 2: ```suggestion
. The default namespace-scoped `PersesDatasource` is in the same namespace as the dashboard.
```

> . The default `PersesGlobalDatasource` available cluster-wide.

Reviewer 2: ```suggestion
. The default `PersesGlobalDatasource` is available cluster-wide.
```
